### PR TITLE
Fix typos found by codespell

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**

### CODE REVIEW

File: dictionary-std/src/uids.rs [hunk 1]
```diff
@@ -1420,10 +1420,10 @@ pub const IMPLANT_TEMPLATE_GROUP_INFORMATION_MODEL_MOVE: &str = "1.2.840.10008.5
 /// SOP Class: Implant Template Group Information Model - GET
 #[rustfmt::skip]
 pub const IMPLANT_TEMPLATE_GROUP_INFORMATION_MODEL_GET: &str = "1.2.840.10008.5.1.4.45.4";
-/// Application Hosting Modle: Native DICOM Model
+/// Application Hosting Model: Native DICOM Model
```
Comment (Enet4) [inline dictionary-std/src/uids.rs:1423]: Looks like the typo leaked into the generated files. It's fine to fix them here, might also think about generating updated files soon.

**Review by Enet4 — COMMENTED** [review]
Fair, some typos slip occasionally and they add up. It's curious that typos such as the one in partial.rs:310 wasn't picked up, were they explicitly excluded because they are already fixed in #557?

**Comment by DimitriPapadopoulos** [comment]
The codespell tool is based on a dictionary of common typos, `retrievies` was not part of that dictionary.

I have added this typo in https://github.com/codespell-project/codespell/pull/3530.

**Review by Enet4 — APPROVED** [review]
Sounds good, thank you. :+1:

**Pull request merged**